tests/rbd/rbd_utils.py

```diff
@@ -276,17 +276,20 @@ def snap_rename(self, pool_name, image_name, current_snap_name, new_snap_name):
             log.error(f"Snapshot rename verification failed for {current_snap_name}")
             return 1
 
-    def snap_remove(self, pool_name, image_name, snap_name):
+    def snap_remove(self, pool_name, image_name, snap_name, **kw):
         """
         Removes a snap of an image in a specified pool name and image name
         Args:
             pool_name  : name of the pool where image is to be imported
             image_name : name of the image file to be imported as
             snap_name  : name of the snapshot
         """
-
-        cmd = f"rbd snap rm {pool_name}/{image_name}@{snap_name}"
-        return self.exec_cmd(cmd=cmd)
+        if kw.get("image_id"):
+            cmd = f"rbd snap rm --image-id {kw.get('image_id')} --pool {pool_name} --snap {snap_name}"
+            return self.exec_cmd(cmd=cmd)
```

Comment:
add this `exec_cmd` at last as it's applicable to both conditions.

Reply:
Updated & done